### ISSUE TITLE
Color contrast ratio is failing for "Docs Link Focus" and respective its background in the "Type chat screen" which is less then less than 3:1 : A11y_TypeChat Website_Type chat_Docs Link_Color Contrast. 

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/TypeChat#229]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #TypeChat Website; #Feature_TypeChat Website_Web_April2024; #Win11; #Color contrast; #WCAG1.4.3; #WCP.

### Environment and OS details:

URL: https://microsoft.github.io/TypeChat/
Windows 11
OS build: 22631.3296

### Repro Steps:

1. Hit the URL: https://microsoft.github.io/TypeChat 
2. Typechat Home screen gets opened.
3. Tab Navigate to "Doc Link" .
4. Turn on Accessibility insights for windows and Observe that Color contrast ratio is failing for "Docs Link Focus" and respective its background in the "Type chat screen" which is less then less than 3:1

### Actual Result:

Color contrast ratio is failing for "Docs Link Focus" and respective its background in the "Type chat screen" which is less then less than 3:1

### Expected Result:

Color contrast ratio for "Docs Link Focus" and respective its background in the "Type chat screen" should be greater than 3:1

### User Impact:

Low vision users will face difficulty if Color contrast ratio is failing for "Docs Link Focus" and respective its background in the "Type chat screen" which is less then less than 3:1.

**Comment by PoojaNamde** [comment]
#Regressed:12-12-24;

Verified this issue in the env: https://microsoft.github.io/[TypeChat](https://microsoft.github.io/TypeChat/)/. Issue is still reproducing.

![Image](https://github.com/user-attachments/assets/9cc6d141-3c90-480c-9f15-14e83c1284f4)

**Comment by AnKushSingh05** [comment]
#A11ySev3;

Verified this issue in the env: [https://microsoft.github.io/[TypeChat](https://microsoft.github.io/TypeChat/)/](https://microsoft.github.io/%5BTypeChat%5D(https://microsoft.github.io/TypeChat/)/)
Downgrading to sev3
